File: 人机交互2020/9-4/9-4/2-2 C#+类/Test/frmTest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using System.IO;

namespace Test
{
    public partial class frmTest : Form
    {
        public frmTest()
        {
            InitializeComponent();
        }

        private void btnSearch_Click(object sender, EventArgs e)
        {
            String path = "D:\\0000人机交互";
            clsDir Dir=new clsDir(path);
            Dir.Output(lstInfo);
        }
    }
}

File: 人机交互2020/9-4/9-4/1-2 VB.net+类/Test/frmTest.vb
Imports System.IO

Public Class frmTest

    Private Sub btnSeach_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSeach.Click
        Dim path As String = "D:\0000人机交互"
        Dim dir As New clsDir(path)
        dir.Output(lstInfo)
    End Sub
End Class
